better_bookmarks/js/content_script.js

```diff
@@ -0,0 +1,28 @@
+var port = chrome.runtime.connect();
+
+window.addEventListener("message", (event) => {
+    if (event.source != window)
+        return;
+
+    if (event.data.type && (event.data.type == "FROM_PAGE")) {
+        console.log("Content script received: " + event.data.text);
+        port.postMessage(event.data.text);
+    }
+}, false);
```

Comment:
I think we can remove everything above this comment, because I think this is only useful when the content script is communicating with the current page.  Verify this though and let me know.

Reply:
Verify whether or not we actually need the above code, and if not, remove it.